community/nodejs-current/APKBUILD

```diff
@@ -1,27 +1,20 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jose-Luis Rivas <ghostbar@riseup.net>
 #
-# secfixes:
-#   9.10.0-r0:
-#     - CVE-2018-7158
-#     - CVE-2018-7159
-#     - CVE-2018-7160
-#   9.2.1-r0:
-#     - CVE-2017-15896
-#     - CVE-2017-15897
-#
 pkgname=nodejs-current
 # The current stable version, i.e. non-LTS.
-pkgver=9.11.1
-pkgrel=4
+
+pkgver=10.10.0
+pkgrel=1
```

Comment:
Node.js v11.1.0 is already out, can you update this PR? AFAIK pgrel should be set to 0